packages/marko-web-inquiry/browser/default-form.vue

```diff
@@ -131,6 +131,14 @@
       </div>
     </div>
     <pre v-if="error" class="alert alert-danger text-danger">An error occurred: {{ error }}</pre>
+    <vue-recaptcha
+      ref="invisibleRecaptcha"
+      size="invisible"
+      :sitekey="sitekey"
+      :load-recaptcha-script="true"
+      @verify="onVerify"
+      @expired="onExpired"
+    />
     <button type="submit" class="btn btn-primary" :disabled="loading">
       Submit
     </button>
```

Comment:
You could simplify a lot of this logic by wrapping the submit button inside the `vue-recaptcha` component itself, as done in the [contact-us form](https://github.com/base-cms-websites/endeavor-business-media/blob/master/packages/contact-us/browser/form.vue#L93). Unless there were major changes in `v1.3.0` of the library, you shouldn't need the `size` or `load-...` params.

This also has the added benefit of allowing you to drop your `onSubmit` method, which is calling an undocumented method in the vue-recaptcha component 😄